sdk/javascript/src/nem/models.js

```diff
@@ -5397,6 +5522,11 @@ class TransferTransaction {
 		this._recipientAddressSize = 40; // reserved field
 	}
 
+	sort() {
+		if (0 !== this.messageEnvelopeSize)
+			this._message.sort();
```

Comment:
this is generated cause message is struct, right?

Reply:
yep